manifests/mod/ssl.pp

```diff
@@ -67,10 +67,25 @@
     'Suse'    => '/var/lib/apache2/ssl_scache(512000)'
   }
 
-  ::apache::mod { 'ssl':
-    package => $package_name,
+  if $::osfamily == 'Suse' {
+    if defined(Class['::apache::mod::worker']){
+      $suse_path = '/usr/lib64/apache2-worker'
+    } elsif defined(Class['::apache::mod::prefork']){
+      $suse_path = '/usr/lib64/apache2-prefork'
+    } else {
+      $suse_path = pick('/usr/lib64/apache2-worker', '/usr/lib64/apache2-prefork')
+    }
+    ::apache::mod { 'ssl':
+      package  => $package_name,
+      lib_path => $suse_path
```

Comment:
technically this resource declaration doesn't need to be duplicated because `lib_path => $suse_path` when undef would be as if it wasn't passed at all